pkg/alertmanager/alertmanager.go

```diff
@@ -97,9 +104,16 @@ func New(cfg *Config, reg *prometheus.Registry) (*Alertmanager, error) {
 		stop:      make(chan struct{}),
 		active:    false,
 		activeMtx: sync.Mutex{},
+		configHashMetric: prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
We also have `cortex_runtime_config_hash` where we expose `sha256` label. Would it work if we do the same here, to keep consistency with that metric?

Reply:
in general I prefer alertmanager's approach, then you could have the label with `count_values("hash", cortex_runtime_config_hash) without()`. Then you can run `changes()` on it.